### ISSUE TITLE
Comment popup is not displayed correctly on some mobile devices

### ISSUE THREAD

**Issue by tim-krieger** [ether/ep_comments_page#192]
Popup on my iPad (2019), Safari mobile, recent version of ep_comments_page, etherpad 1.8.12:

![image](https://user-images.githubusercontent.com/64839715/110393764-676d3b80-806b-11eb-8693-c9dc76271919.png)

**Comment by JohnMcLear** [comment]
@seballot any idea how this can happen?

**Comment by seballot** [comment]
Yep : https://github.com/ether/ep_comments_page/blob/master/static/js/newComment.js#L85

The popup is aligned with comment icon, which does not work when comment icon is closed to the right border of the screen. Should be easy fix

**Comment by tim-krieger** [comment]
I think my iPad is not recognized as media only screen (800px). In web inspector the popup is displayed correctly. Should this be fixed in ether/etherpad-lite? It also infects other plugins with toolbar popups like ep_embedded_hyperlinks2.
![image](https://user-images.githubusercontent.com/64839715/124866932-83eb9700-dfbd-11eb-89d8-2dd38c6a1ae0.png)

**Comment by tim-krieger** [comment]
If I use max-width 992px instead of 800px in popup.css (etherpad-lite) the new comment popup is diplayed correctly (iPad 2020).
